frontend/app/(defaultLayout)/Troubleshooting/[device]/components/NavBar.tsx

```diff
@@ -0,0 +1,200 @@
+                     },
+                     '::after': {
+                        right: '0',
+                        background:
+                           'linear-gradient(to left, #fff 60%, rgba(255, 255, 255, 0))',
+                     },
+                  }}
+                  position="relative"
+                  flex="1 2"
+                  overflowX="auto"
+               >
+                  <NavTabs
+                     overflowX="auto"
+                     flexGrow="1"
+                     paddingInline={{ base: 0, sm: 2 }}
+                     deviceGuideUrl={deviceGuideUrl}
+                     devicePartsUrl={devicePartsUrl}
+                  />
+               </Box>
+            </Flex>
+         </Flex>
+      </Flex>
+   );
+}
+
+type NavTabsProps = {
+   deviceGuideUrl?: string;
+   devicePartsUrl?: string;
+};
+
+function NavTabs({
+   devicePartsUrl,
+   deviceGuideUrl,
+   ...props
+}: NavTabsProps & FlexProps) {
+   // The type here works because all the styles we want to use are available on
+   // both Box and Link
+   const baseStyleProps: BoxProps & LinkProps = {
+      outline: '2px solid transparent',
+      outlineOffset: '2px',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      paddingBlock: 2,
+      paddingInline: 4,
+      position: 'relative',
+   };
+
+   const bottomFeedbackStyleProps = {
+      content: '""',
+      position: 'absolute',
+      bottom: 0,
+      left: 0,
+      right: 0,
+      height: '3px',
+      borderRadius: '2px 2px 0px 0px',
+   };
+
+   const selectedStyleProps = {
+      ...baseStyleProps,
+      borderColor: 'blue.500',
+      color: 'gray.900',
+      fontWeight: 'medium',
+      _visited: {
+         color: 'gray.900',
+      },
+      _hover: {
+         textDecoration: 'none',
+         background: 'gray.100',
+         '::after': {
+            background: 'blue.700',
+         },
+      },
+      _after: {
+         ...bottomFeedbackStyleProps,
+         background: 'blue.500',
+      },
+   };
+
+   const notSelectedStyleProps = {
+      ...baseStyleProps,
+      borderColor: 'transparent',
+      color: 'gray.500',
+      fontWeight: 'normal',
+      _hover: {
+         textDecoration: 'none',
+      },
+      _visited: {
+         color: 'gray.500',
+      },
+      sx: {
+         '&:hover:not(.isDisabled)': {
+            color: 'gray.700',
+            background: 'gray.100',
+         },
+         '&.isDisabled': {
+            opacity: 0.4,
+            cursor: 'not-allowed',
+            color: 'gray.500',
+         },
+      },
+   };
+
+   return (
+      <Flex {...props} gap={1.5} height="100%">
+         {devicePartsUrl ? (
+            <Link {...notSelectedStyleProps} href={devicePartsUrl}>
+               Parts
+            </Link>
+         ) : (
+            <Box className="isDisabled" {...notSelectedStyleProps}>
+               Parts
+            </Box>
```

Comment:
Just a thought on design: Should we add title text to the disabled button explaining that there are no parts / guides? Otherwise the user might think something is just broken if we are showing a disabled button with no option to use it.

Reply:
I would hope that users understand the disabled state. If we were to do anything, we could consider not showing those at all if there aren't any results. I have a UI issue queued up at https://github.com/iFixit/ifixit/issues/51698 - let's pick up this conversation there